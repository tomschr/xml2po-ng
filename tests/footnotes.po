msgid ""
msgstr ""
"Project-Id-Version: footnotes test\n"
"POT-Creation-Date: 2006-03-31 21:30+0200\n"
"PO-Revision-Date: 2006-03-31 21:33+0200\n"
"Last-Translator: Данило\n"
"Language-Team: Serbian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: footnotes.xml:6(title)
msgid "Introduction"
msgstr "Увод"

#: footnotes.xml:9(para)
msgid "Those short explanations on the bottom of the page"
msgstr "Она кратка објашњења у дну стране"

#: footnotes.xml:8(para)
msgid "This is a test of footnotes<placeholder-1/> extraction, since footnotes are not \"final\" (i.e. directly extracted) elements."
msgstr "Ово је проба извлачења фуснота<placeholder-1/>, пошто фусноте нису „финални“ (тј. директно извучени) елементи."

#: footnotes.xml:15(para)
msgid "Remember?"
msgstr "Сећаш се?"

#: footnotes.xml:13(para)
msgid "That means that if there is some ignorable content inside the footnote<placeholder-1/>, they would still not be shown in the output. This also applies to other cases of nested tags, but it was originally reported against footnotes as bug #172784."
msgstr "Ово значи да ако има нешто занемарљивог садржаја унутар фусноте<placeholder-1/>, оне се и даље неће видети у излазу. Ово такође важи и за остале случајеве угњеждених тагова, али је првобитно пријављено за фусноте као грешка #172784."

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: footnotes.xml:0(None)
msgid "translator-credits"
msgstr ""
