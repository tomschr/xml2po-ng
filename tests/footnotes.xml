<?xml version="1.0"?>
<!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN"
    "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" >
<article>
  <sect1 id="intro">
	<title>Introduction</title>

	<para>
	  This is a test of footnotes<footnote><para>Those short
          explanations on the bottom of the page</para></footnote> extraction,
          since footnotes are not "final" (i.e. directly extracted) elements.
        </para>
	<para>
	  That means that if there is some ignorable content inside the footnote<footnote>
           <para>Remember?</para>
           <!-- nothing important -->
          </footnote>, they would still not be shown in the output.
           This also applies to other cases of nested tags, but it was
           originally reported against footnotes as bug #172784.
        </para>
  </sect1>

</article>
