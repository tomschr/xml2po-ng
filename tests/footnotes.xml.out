<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd">
<article lang="footnotes">
  <sect1 id="intro">
	<title>Увод</title>

	<para>Ово је проба извлачења фуснота<footnote><para>Она кратка објашњења у дну стране</para></footnote>, пошто фусноте нису „финални“ (тј. директно извучени) елементи.</para>
	<para>Ово значи да ако има нешто занемарљивог садржаја унутар фусноте<footnote>
           <para>Сећаш се?</para>

          </footnote>, оне се и даље неће видети у излазу. Ово такође важи и за остале случајеве угњеждених тагова, али је првобитно пријављено за фусноте као грешка #172784.</para>
  </sect1>

</article>
